requirements.txt

```diff
@@ -2,3 +2,8 @@ numpy==1.16.*
 pandas==0.24.*
 pylabrad==0.97.*
 pytest==4.2.*
+matplotlib
+lmfit
+sympy
+scipy
+inflect
```

Comment:
We typically sort imports, and it's almost always a good idea to pin the versions here. Otherwise you'll have person A install version 1.1.0 and person B will install 1.2.0 and run into problems. Usually we do `major.minor.*` to ensure we install the latest security and bug fixes, but with a stable API :)

Reply:
Thank you Kunal! Made the changes.